db/index.js

```diff
@@ -0,0 +1,29 @@
+import { Sequelize } from 'sequelize';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const { DB_NAME, DB_USER, DB_PASSWORD, DB_HOST, DB_PORT } = process.env;
+
+export const sequelize = new Sequelize(DB_NAME, DB_USER, DB_PASSWORD, {
+  host: DB_HOST,
+  port: DB_PORT,
+  dialect: 'postgres',
+  dialectOptions: {
+    ssl: {
+      require: true,
+      rejectUnauthorized: false,
+    },
+  },
+  logging: false,
+});
+
+export const connectDB = async () => {
+  try {
+    await sequelize.authenticate();
+    console.log('Database connection successful');
+  } catch (error) {
+    console.error('Unable to connect to the database:', error.message);
+    process.exit(1);
```

Comment:
```suggestion
    throw new Error("Failed to connect to DB: " + err.message);
```
So we can detect the place it crashed and its callstack(process.exit() doesn't show such info).